en/config_rover/ackermann.md

```diff
@@ -1,21 +1,18 @@
-# Ackermann Rover
+# Configuration/Tuning (Ackermann Rover)
```

Comment:
This "might" do with a restructure similar to the other (if that makes sense). Good enough for now though.